### ISSUE TITLE
title for AZ numeral section

### ISSUE THREAD

**Issue by KenK-3-21** [apertium/apertium-aze#1]
%<det%>        ! Determiner  I wonder if this shouldn't be  %<clf%>  ! Classifier

**Comment by ftyers** [comment]
That's for stuff like "bir" (in "güzel bir kız") and "bu" ... I'm not sure if I'd call "bu" a classifier. @jonorthwash what do you think? If I were to label anything classifier it would probably be stuff like "tane" maybe?

**Comment by jonorthwash** [comment]
Agreed.

Words "bu" are both [determiners](https://en.wikipedia.org/wiki/Determiner) and pronouns.  [Classifiers](https://en.wikipedia.org/wiki/Classifier_(linguistics)) are normally something else, cf. what @ftyers was thinking.  This will of course depend on terminology, but most linguistics terminology uses these words in this way.

So this raises the question, @KenK-3-21: what do you mean by classifier?  Perhaps "classifier" in your terminology maps to "determiner"/"pronoun" in Apertium terminology.  It's also possible that "bu" has uses in `aze` that we're not aware of, so demonstration of it matching the Apertium community's understanding of "classifier" would simply need some demonstration of its distribution matching that term.

**Comment by KenK-3-21** [comment]
Hi gentlemen,

By classifier, I mean words like:

dana, ədəd, nəfər

Üç dana kitab
On ədəd yumurta
Bir nəfər

In Chinese, these are called “量詞”.

Cheers,

Ken

**Comment by jonorthwash** [comment]
Yes, those words should probably be called classifiers, with `<clf>`.  Are they called `<det>` somewhere?

**Comment by KenK-3-21** [comment]
Hi,

I'm sorry, it didn't occur to me to think of "bir" as a determiner.  So
that's why I am proposing <clf> as a new category.

It's difficult for me to understand how "bu" would function as a
determiner. Can you give me an example? I would think that <dem> to be a
more appropriate category.

Unfortunately, Azeri only has two demonstratives..."bu", and "o", in
contrast to Uzbek and Uighur, which have three demonstratives. "o", "bu"
and "şu".

Cheers, Ken

**Comment by KenK-3-21** [comment]
Sorry, I should have written u, bu, and şu for Uighur-Uzbek. KK

**Comment by jonorthwash** [comment]
> I'm sorry, it didn't occur to me to think of "bir" as a determiner. So that's why I am proposing <clf> as a new category.

This is two separate issues.  The non-number use of "bir" can be a determiner (indefinite, like "a" in English), and it's different from words like "dana" and "ədəd".  Creating a category of `<clf>` for these words is quite reasonable, and probably is the right thing to do.

> It's difficult for me to understand how "bu" would function as a determiner. Can you give me an example? I would think that <dem> to be a more appropriate category.

We consider "bu" to be a demonstrative determiner (`<det><dem>`, where `<dem>` is the subcategory, in opposition to e.g. interrogative determiners) and also a demonstrative pronoun (`<prn><dem>`, in opposition to e.g. personal pronouns).

Here are a couple examples in Turkish / Uzbek demonstrating the two uses:

* Bu benim evim.  /  Bu mening uyim.  (pronoun)
* Bu ev benimki.  /  Bu uy meniki.  (determiner)

So while both uses of "bu" are demonstrative, we're concerned with the syntactic patterning more than the semantics—this is why we divide the two into determiner and pronoun.

> Unfortunately, Azeri only has two demonstratives..."bu", and "o"

I'm not sure why this is unfortunate.  It doesn't really make much difference from the point of view of morphological analysis and generation—so you have two forms in the dictionary instead of three, which isn't a big deal.

**Comment by KenK-3-21** [comment]
Hi Jonathan,

I see your reasoning as to why you would include o/bu <dem> as a
subcategory of the class determiner <det>. That makes sense.

As to considering the state of Western Turkic's relative lack of
demonstratives, I guess I just don't like my choices to be limited. Having
an intermediate distal demonstrative seems more enriching.

Just kidding.

KK
